sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -454,6 +454,7 @@ public partial class RetryOptions
     public abstract partial class TokenCredential
     {
         protected TokenCredential() { }
+        public virtual bool SupportsCaching { get { throw null; } }
```

Comment:
Should we update all credentials which use MSAL for authentication to override this to return true? Also since this will be in a GA version of Core do you think we should patch Azure.Identity 1.4.0 to properly implement these for all our credentials?

Reply:
Good call out - I'm not sure if we need to patch though considering that all of them are already operating as if they return false for this value. But perhaps there is a scenario I am missing here.